testfiles/CMakeLists.txt

```diff
@@ -141,6 +141,7 @@ add_simulation_test(IDF_FILE AirflowNetwork_Simple_SmallOffice.idf EPW_FILE USA_
 add_simulation_test(IDF_FILE AirflowWindowsAndBetweenGlassBlinds.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
 add_simulation_test(IDF_FILE AirflowWindowsAndBetweenGlassShades.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
 add_simulation_test(IDF_FILE ASIHPMixedTank.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+add_simulation_test(IDF_FILE AtticRoof_RadiantBarriers.idf EPW_FILE USA_AZ_Phoenix-Sky.Harbor.Intl.AP.722780_TMY3.epw)
```

Comment:
Thanks for adding the new test file, once it is running well this will be a great exercise of the algorithm.